client/src/styles/common.scss

```diff
@@ -25,6 +25,9 @@ h2 {
 
   &[disabled] {
     color: var(--text-color-disabled);
+    background-color: transparent;
+    border: 1px solid var(--overlay-bg-color);
```

Comment:
why you didn't use scss constants? @Pr1sm0 @N-Balinskiy @AmbeMO

Reply:
Resolved